TTS/tts/layers/xtts/stream_generator.py

```diff
@@ -24,7 +24,6 @@
 from transformers.generation.stopping_criteria import validate_stopping_criteria
 from transformers.generation.utils import GenerateOutput, SampleOutput, logger
 
-
```

Comment:
Could you add back this blank line? Otherwise the style check fails (you can run `make style` locally to check).

Reply:
@gravityrail Are you planning to make these changes?

Reply:
Just fixed it myself, should be good to go now